helm/cluster-cloud-director/values.yaml

```diff
@@ -1,3 +1,9 @@
+apiServer:
+  enableAdmissionPlugins: "NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota,PersistentVolumeClaimResize,DefaultStorageClass,Priority,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook"
+  featureGates: "TTLAfterFinished=true"
+controllerManager:
+  featureGates: "ExpandPersistentVolumes=true,TTLAfterFinished=true"
```

Comment:
I wonder if we really need to expose that in `values.yaml`.

Reply:
As far as I remember, while implementing these flags for CAPO, @pipo02mix  said we need to test a feature flag from time to time and it is better to have an interface to customize them.